### ISSUE TITLE
Margin for Negative Pair

### ISSUE THREAD

**Issue by zhumanli** [dimitrismallis/KeypointsToLandmarks#3]
Hi,

Thank you very much again for your answers to my previous questions.

I'd like to know how the margin for negative pairs is selected, I see you have used 0.8 for the margin value in your source codes, did you do any tuning for this parameter?

And how do you compare the loss used in this paper and the Normalized Temperature-Scaled Cross-Entropy Loss (NT-Xent loss) used in the SimCLR paper?

Thank you so much for your valuable time.

**Comment by dimitrismallis** [comment]
Hi,

Indeed we use a margin of 0.8 for all the experiments reported in this work. We found that this value performed well without exhaustively tuning this parameter.

Regarding the loss, we use a standard max margin contrastive loss (for example [1]). NT-Xent is a different loss function with different properties. For example, it measures cosine similarity whereas our loss measures euclidean distances. We did not experiment with different loss functions like the NT-Xent loss you mention. 

[1] Choy, C. B., Gwak, J., Savarese, S., & Chandraker, M. (2016). Universal correspondence network. Advances in neural information processing systems, 29.
